services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudV2Spec.groovy

```diff
@@ -494,6 +494,21 @@ class FlowCrudV2Spec extends HealthCheckSpecification {
         ]
     }
 
+    def "Unable to create a flow with invalid encapsulation type"() {
```

Comment:
What about a value with lower/upper/mixed casa for encapsulation type?

Reply:
Func tests with such cases were rejected by QA team